src/sortStudents.ts

```diff
@@ -1,16 +1,62 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
```

Comment:
In `enam` better write this way, it would be more obvious(all over the enam key-values).
```suggestion
  Name = 'Name', 
```
or
```suggestion
  Name = 'name', 
```
or 
```suggestion
  Name = 'NAME', 
```